NEWS.md

```diff
@@ -0,0 +1,29 @@
+# 3.2.3 / 2018-06-24
+* No user visible changes
+
+# 3.2.2 / 2018-06-23
+* Add keystone v3 support
+* Add missing nova utility methods for server actions
+* Add heat support
+* Add router support
+* All fixes and improvements up to 3.1.3
+
```

Comment:
What happened to 3.2.0 and 3.2.1?

Reply:
I added two small things to 3.2.0. 3.2.0 and 3.2.1 just differ in version number, looks like a change in the release tolling.